toolchain/check/context.cpp

```diff
@@ -144,6 +144,26 @@ auto Context::PopScope() -> void {
   }
 }
 
+auto Context::SkipNameReferences(SemIR::NodeId node_id) -> SemIR::NodeId {
+  while (true) {
```

Comment:
(super trivial comment)

Slightly shorter?

```suggestion
  for (;;) {
```

Or do we have an established idiom for infinite loops? I don't feel strongly.

Reply:
We have about a dozen `while (true)` and zero `for (;;)` in C++ code, one of each in `docs/`, and one of each in `proposals/` (the `for (;;)` in docs and proposals are both pointing out that that's a C++ thing that doesn't exist in that form in Carbon).

Reply:
🤷🏻

Reply:
I think we should do the thing that is available in Carbon, and if it seems painful, use that as a signal that we should address that in Carbon's design.